test/helpers/fake-project.js

```diff
@@ -87,6 +87,16 @@ module.exports = class FakeProject extends FixturifyProject {
     this.writeSync();
   }
 
+  writeLintTodo(daysToDecay) {
```

Comment:
This seems like an odd function name, it isn't about writing a todo its about writing lintTodoConfig right?

Reply:
`lintTodo` is the name of the key in the package.json file. So, it is writing a `lintTodo`. 🤷‍♂️

Reply:
Updated.